students/eowyn/session13/raises.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+"""
+detect whether correct exception was thrown otherwise raise it
+"""
+
+## would like to throw specific error
+
+class raises():
+
+    def __init__(self, ErrType):
+        self.ErrType = ErrType
+        pass
```

Comment:
no need for a pass here. pass if for when there is literally nothing in a method.